libs/shared/src/lib/models/search/search-kind.enum.ts

```diff
@@ -0,0 +1,8 @@
+export enum SearchKind {
```

Comment:
I don't think having multiple selection is the right move. In particular, if you want to stop searching for prose and poetry and only search for users, then requires both unselecting prose and poetry and selecting users. If we display multiple kinds on the search results, that causes complications because they use different cards and, if they're displayed separately, then figuring out pagination could result in complications. I think it makes more sense to just display results for one kind (and Prose and Poetry), which also matches how it is on other sites.